tensorflow/python/ops/nn_ops.py

```diff
@@ -1294,7 +1294,7 @@ def conv3d_transpose(value,
     padding: A string, either `'VALID'` or `'SAME'`. The padding algorithm.
       See the @{tf.nn.convolution$comment here}
     data_format: A string, either `'NDHWC'` or `'NCDHW`' specifying the layout
-      of the input and output tensors. Defaults to `'NDHWC'`.
```

Comment:
@martinwicke I did a rebase on master at some point of time, data_format is `None` by default, just like other method signatures. But the thing is, it acts like `NDHWC` being a NoneType, if not `NCDHW`. I modified the docstring because the default value is not `NDHWC` explicitly.